db/migrate/20160426212643_fix_step_user_completer.rb

```diff
@@ -0,0 +1,36 @@
+class FixStepUserCompleter < ActiveRecord::Migration
+  class Step < ActiveRecord::Base
+    belongs_to :user
+    belongs_to :completer, class_name: "User"
+  end
+
+  class User < ActiveRecord::Base
+    has_many :outgoing_delegations, class_name: "UserDelegate", foreign_key: "assigner_id"
+    has_many :incoming_delegations, class_name: "UserDelegate", foreign_key: "assignee_id"
+  end
+
+  class UserDelegate < ActiveRecord::Base
+    belongs_to :assignee, class_name: "User", foreign_key: "assignee_id"
+    belongs_to :assigner, class_name: "User", foreign_key: "assigner_id"
+  end
+
+  def up
+    steps_with_no_completer.each do |step|
```

Comment:
seems like this could live in a one-off rake task rather than migration since we just need to run it once.